### DISCUSSION TITLE
Transferring the mod to 1.16.5

### DISCUSSION THREAD

**Discussion by porovozik7767** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1147] [Q&A]
Hello, I really wanted the mod to be on version 1.16.5.
It's possible to somehow transfer it to version 1.16.5 I want to play with friends with this mod. And 1.7 is A VERY old version..
I will be very glad if there is a solution to this problem. (There may be mistakes in the text and words that are not clear, since I am Russian, I translate the text through a translator.)

**Comment by JameH2** [comment]
you opened this post before?

**Reply by porovozik7767** [reply]
I wrote about the updates from the author, and now how to redo it myself on 1.16.5

**Comment by BallOfEnergy1** [comment]
There are no plans to port 1.7.10 HBM NTM to a newer version, though there are individual forks that port to newer versions. You can create your own if you feel like porting it.
